File: src/ReservationBundle/Entity/Reservation.php
<?php

namespace ReservationBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Reservation
 *
 * @ORM\Table(name="Reservation")
 * @ORM\Entity(repositoryClass="ReservationBundle\Repository\ReservationRepository")
 */
class Reservation
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_Reservation", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idReservation;

    /**
     * @var int
     * @ORM\ManyToOne(targetEntity=AppBundle\Entity\Client::class)
     * @ORM\JoinColumn(name="id_client" , referencedColumnName="id")
     */
    private $idClient;

    /**
     * @var int
     * @ORM\ManyToOne(targetEntity=AppBundle\Entity\chauffeur::class)
     * @ORM\JoinColumn(name="id_chauffeur" , referencedColumnName="id")
     */
    private $idChauffeur;

    /**
     * @var string
     *
     * @ORM\Column(name="depart", type="string", length=255, nullable=true)
     */
    private $depart;

    /**
     * @var string
     *
     * @ORM\Column(name="arrive", type="string", length=255, nullable=true)
     */
    private $arrive;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="heure", type="datetime", options={"default": "CURRENT_TIMESTAMP"})
     */
    private $heure;

    /**
     * @var float
     *
     * @ORM\Column(name="prix", type="float", precision=10, scale=0, nullable=false)
     */
    private $prix;

    /**
     * @var string
     *
     * @ORM\Column(name="type_Reservation", type="string", length=255, nullable=false)
     */
    private $typeReservation;

    /**
     * @var float
     *
     * @ORM\Column(name="latitude", type="float", precision=10, scale=0, nullable=false)
     */
    private $latitude;

    /**
     * @var float
     *
     * @ORM\Column(name="longitude", type="float", precision=10, scale=0, nullable=false)
     */
    private $longitude;

    /**
     * @var float
     *
     * @ORM\Column(name="latitude2", type="float", precision=10, scale=0, nullable=false)
     */
    private $latitude2;

    /**
     * @var float
     *
     * @ORM\Column(name="longitude2", type="float", precision=10, scale=0, nullable=false)
     */
    private $longitude2;

    /**
     * @return int
     */
    public function getIdReservation()
    {
        return $this->idReservation;
    }

    /**
     * @param int $idReservation
     */
    public function setIdReservation($idReservation)
    {
        $this->idReservation = $idReservation;
    }

    /**
     * @return int
     */
    public function getIdClient()
    {
        return $this->idClient;
    }

    /**
     * @param int $idClient
     */
    public function setIdClient($idClient)
    {
        $this->idClient = $idClient;
    }

    /**
     * @return int
     */
    public function getIdChauffeur()
    {
        return $this->idChauffeur;
    }

    /**
     * @param int $idChauffeur
     */
    public function setIdChauffeur($idChauffeur)
    {
        $this->idChauffeur = $idChauffeur;
    }

    /**
     * @return string
     */
    public function getDepart()
    {
        return $this->depart;
    }

    /**
     * @param string $depart
     */
    public function setDepart($depart)
    {
        $this->depart = $depart;
    }

    /**
     * @return string
     */
    public function getArrive()
    {
        return $this->arrive;
    }

    /**
     * @param string $arrive
     */
    public function setArrive($arrive)
    {
        $this->arrive = $arrive;
    }

    /**
     * @return \DateTime
     */
    public function getHeure()
    {
        return $this->heure;
    }

    /**
     * @param \DateTime $heure
     */
    public function setHeure($heure)
    {
        $this->heure = $heure;
    }

    /**
     * @return float
     */
    public function getPrix()
    {
        return $this->prix;
    }

    /**
     * @param float $prix
     */
    public function setPrix($prix)
    {
        $this->prix = $prix;
    }

    /**
     * @return string
     */
    public function getTypeReservation()
    {
        return $this->typeReservation;
    }

    /**
     * @param string $typeReservation
     */
    public function setTypeReservation($typeReservation)
    {
        $this->typeReservation = $typeReservation;
    }

    /**
     * @return float
     */
    public function getLatitude()
    {
        return $this->latitude;
    }

    /**
     * @param float $latitude
     */
    public function setLatitude($latitude)
    {
        $this->latitude = $latitude;
    }

    /**
     * @return float
     */
    public function getLongitude()
    {
        return $this->longitude;
    }

    /**
     * @param float $longitude
     */
    public function setLongitude($longitude)
    {
        $this->longitude = $longitude;
    }

    /**
     * @return float
     */
    public function getLatitude2()
    {
        return $this->latitude2;
    }

    /**
     * @param float $latitude2
     */
    public function setLatitude2($latitude2)
    {
        $this->latitude2 = $latitude2;
    }

    /**
     * @return float
     */
    public function getLongitude2()
    {
        return $this->longitude2;
    }

    /**
     * @param float $longitude2
     */
    public function setLongitude2($longitude2)
    {
        $this->longitude2 = $longitude2;
    }


}

File: SprintMobiles/src/com/bolt/app/entities/Reservation.java
<<<<<<< HEAD
<<<<<<< HEAD
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.bolt.app.entities;

import java.util.Objects;

/**
 *
 * @author Armand
 */
public class Reservation {
    private int id_reservation;
    private Client client;
    private Chauffeur chauffeur;
    private String depart;
    private String arrive;
    private double prix;
    private String type_reservation;
    private int code_liv;
    private int nbr_place;
    private double latitude;
    private double longitude;
    private double latitude2;
    private double longitude2;

    public Reservation() {
    }

    public Reservation(Client client, Chauffeur chauffeur, String depart, String arrive,double prix, String type_reservation, int code_liv) {
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = "livraison";
        this.code_liv = code_liv;
    }

   public Reservation(Client client, Chauffeur chauffeur, String depart, String arrive,  double prix, int nbrplace) {
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = "covoiturage";
        this.nbr_place = nbrplace;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    public double getLatitude2() {
        return latitude2;
    }

    public void setLatitude2(double latitude2) {
        this.latitude2 = latitude2;
    }

    public double getLongitude2() {
        return longitude2;
    }

    public void setLongitude2(double longitude2) {
        this.longitude2 = longitude2;
    }

    public Reservation(int id_reservation, Client client, Chauffeur chauffeur, String depart, String arrive, double prix, String type_reservation, double latitude, double longitude, double latitude2, double longitude2) {
        this.id_reservation = id_reservation;
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = type_reservation;
        this.latitude = latitude;
        this.longitude = longitude;
        this.latitude2 = latitude2;
        this.longitude2 = longitude2;
    }

    public Reservation(Client client, Chauffeur chauffeur, String depart, String arrive, double prix, String type_reservation, double latitude, double longitude, double latitude2, double longitude2) {
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = type_reservation;
        this.latitude = latitude;
        this.longitude = longitude;
        this.latitude2 = latitude2;
        this.longitude2 = longitude2;
    }
   


    public int getId_reservation() {
        return id_reservation;
    }

    public void setId_reservation(int id_reservation) {
        this.id_reservation = id_reservation;
    }

    public Client getClient() {
        return client;
    }

    public void setClient(Client client) {
        this.client = client;
    }

    public Chauffeur getChauffeur() {
        return chauffeur;
    }

    public void setChauffeur(Chauffeur chauffeur) {
        this.chauffeur = chauffeur;
    }

    public String getDepart() {
        return depart;
    }

    public void setDepart(String depart) {
        this.depart = depart;
    }

    public String getArrive() {
        return arrive;
    }

    public void setArrive(String arrive) {
        this.arrive = arrive;
    }

    public double getPrix() {
        return prix;
    }

    public void setPrix(double prix) {
        this.prix = prix;
    }

    public String getType_reservation() {
        return type_reservation;
    }

    public void setType_reservation(String type_reservation) {
        this.type_reservation = type_reservation;
    }

    public int getCode_liv() {
        return code_liv;
    }

    public void setCode_liv(int code_liv) {
        this.code_liv = code_liv;
    }

    public int getNbr_place() {
        return nbr_place;
    }

    public void setNbr_place(int nbr_place) {
        this.nbr_place = nbr_place;
    }

    @Override
    public String toString() {
        String mp= "Reservation{" + "id_reservation=" + id_reservation + ", " + client.toString() + ", " + chauffeur.toString() + ", depart=" + depart + ", arrive=" + arrive + ", prix=" + prix + ", type_reservation=" + type_reservation;
        if(code_liv != 0){
             mp += ", code_liv=" + code_liv + ", nbr_place=" + nbr_place + '}';
        }
           if(type_reservation.equals("covoiturage")){
             mp += ", nbr_place=" + nbr_place + '}';
        }
           
           return mp;
    }

    @Override
    public int hashCode() {
        int hash = 5;
        hash = 61 * hash + this.id_reservation;
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Reservation other = (Reservation) obj;
        if (this.client.getId_user() != other.client.getId_user()) {
            return false;
        }
        if (!Objects.equals(this.depart, other.depart)) {
            return false;
        }
        if (!Objects.equals(this.arrive, other.arrive)) {
            return false;
        }
        if (!Objects.equals(this.type_reservation, other.type_reservation)) {
            return false;
        }
        return true;
    }
    
    
    
}
=======
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.bolt.app.entities;

import java.util.Objects;

/**
 *
 * @author Armand
 */
public class Reservation {
    private int id_reservation;
    private Client client;
    private Chauffeur chauffeur;
    private String depart;
    private String arrive;
    private double prix;
    private String type_reservation;
    private int code_liv;
    private int nbr_place;
    private double latitude;
    private double longitude;
    private double latitude2;
    private double longitude2;

    public Reservation() {
    }

    public Reservation(Client client, Chauffeur chauffeur, String depart, String arrive,double prix, String type_reservation, int code_liv) {
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = "livraison";
        this.code_liv = code_liv;
    }

   public Reservation(Client client, Chauffeur chauffeur, String depart, String arrive,  double prix, int nbrplace) {
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = "covoiturage";
        this.nbr_place = nbrplace;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    public double getLatitude2() {
        return latitude2;
    }

    public void setLatitude2(double latitude2) {
        this.latitude2 = latitude2;
    }

    public double getLongitude2() {
        return longitude2;
    }

    public void setLongitude2(double longitude2) {
        this.longitude2 = longitude2;
    }

    public Reservation(int id_reservation, Client client, Chauffeur chauffeur, String depart, String arrive, double prix, String type_reservation, double latitude, double longitude, double latitude2, double longitude2) {
        this.id_reservation = id_reservation;
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = type_reservation;
        this.latitude = latitude;
        this.longitude = longitude;
        this.latitude2 = latitude2;
        this.longitude2 = longitude2;
    }

    public Reservation(Client client, Chauffeur chauffeur, String depart, String arrive, double prix, String type_reservation, double latitude, double longitude, double latitude2, double longitude2) {
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = type_reservation;
        this.latitude = latitude;
        this.longitude = longitude;
        this.latitude2 = latitude2;
        this.longitude2 = longitude2;
    }
   


    public int getId_reservation() {
        return id_reservation;
    }

    public void setId_reservation(int id_reservation) {
        this.id_reservation = id_reservation;
    }

    public Client getClient() {
        return client;
    }

    public void setClient(Client client) {
        this.client = client;
    }

    public Chauffeur getChauffeur() {
        return chauffeur;
    }

    public void setChauffeur(Chauffeur chauffeur) {
        this.chauffeur = chauffeur;
    }

    public String getDepart() {
        return depart;
    }

    public void setDepart(String depart) {
        this.depart = depart;
    }

    public String getArrive() {
        return arrive;
    }

    public void setArrive(String arrive) {
        this.arrive = arrive;
    }

    public double getPrix() {
        return prix;
    }

    public void setPrix(double prix) {
        this.prix = prix;
    }

    public String getType_reservation() {
        return type_reservation;
    }

    public void setType_reservation(String type_reservation) {
        this.type_reservation = type_reservation;
    }

    public int getCode_liv() {
        return code_liv;
    }

    public void setCode_liv(int code_liv) {
        this.code_liv = code_liv;
    }

    public int getNbr_place() {
        return nbr_place;
    }

    public void setNbr_place(int nbr_place) {
        this.nbr_place = nbr_place;
    }

    @Override
    public String toString() {
        String mp= "Reservation{" + "id_reservation=" + id_reservation + ", " + client.toString() + ", " + chauffeur.toString() + ", depart=" + depart + ", arrive=" + arrive + ", prix=" + prix + ", type_reservation=" + type_reservation;
        if(code_liv != 0){
             mp += ", code_liv=" + code_liv + ", nbr_place=" + nbr_place + '}';
        }
           if(type_reservation.equals("covoiturage")){
             mp += ", nbr_place=" + nbr_place + '}';
        }
           
           return mp;
    }

    @Override
    public int hashCode() {
        int hash = 5;
        hash = 61 * hash + this.id_reservation;
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Reservation other = (Reservation) obj;
        if (this.client.getId_user() != other.client.getId_user()) {
            return false;
        }
        if (!Objects.equals(this.depart, other.depart)) {
            return false;
        }
        if (!Objects.equals(this.arrive, other.arrive)) {
            return false;
        }
        if (!Objects.equals(this.type_reservation, other.type_reservation)) {
            return false;
        }
        return true;
    }
    
    
    
}
>>>>>>> master
=======
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.bolt.app.entities;

import java.util.Objects;

/**
 *
 * @author Armand
 */
public class Reservation {
    private int id_reservation;
    private Client client;
    private Chauffeur chauffeur;
    private String depart;
    private String arrive;
    private double prix;
    private String type_reservation;
    private int code_liv;
    private int nbr_place;
    private double latitude;
    private double longitude;
    private double latitude2;
    private double longitude2;

    public Reservation() {
    }

    public Reservation(Client client, Chauffeur chauffeur, String depart, String arrive,double prix, String type_reservation, int code_liv) {
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = "livraison";
        this.code_liv = code_liv;
    }

   public Reservation(Client client, Chauffeur chauffeur, String depart, String arrive,  double prix, int nbrplace) {
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = "covoiturage";
        this.nbr_place = nbrplace;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    public double getLatitude2() {
        return latitude2;
    }

    public void setLatitude2(double latitude2) {
        this.latitude2 = latitude2;
    }

    public double getLongitude2() {
        return longitude2;
    }

    public void setLongitude2(double longitude2) {
        this.longitude2 = longitude2;
    }

    public Reservation(int id_reservation, Client client, Chauffeur chauffeur, String depart, String arrive, double prix, String type_reservation, double latitude, double longitude, double latitude2, double longitude2) {
        this.id_reservation = id_reservation;
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = type_reservation;
        this.latitude = latitude;
        this.longitude = longitude;
        this.latitude2 = latitude2;
        this.longitude2 = longitude2;
    }

    public Reservation(Client client, Chauffeur chauffeur, String depart, String arrive, double prix, String type_reservation, double latitude, double longitude, double latitude2, double longitude2) {
        this.client = client;
        this.chauffeur = chauffeur;
        this.depart = depart;
        this.arrive = arrive;
        this.prix = prix;
        this.type_reservation = type_reservation;
        this.latitude = latitude;
        this.longitude = longitude;
        this.latitude2 = latitude2;
        this.longitude2 = longitude2;
    }
   


    public int getId_reservation() {
        return id_reservation;
    }

    public void setId_reservation(int id_reservation) {
        this.id_reservation = id_reservation;
    }

    public Client getClient() {
        return client;
    }

    public void setClient(Client client) {
        this.client = client;
    }

    public Chauffeur getChauffeur() {
        return chauffeur;
    }

    public void setChauffeur(Chauffeur chauffeur) {
        this.chauffeur = chauffeur;
    }

    public String getDepart() {
        return depart;
    }

    public void setDepart(String depart) {
        this.depart = depart;
    }

    public String getArrive() {
        return arrive;
    }

    public void setArrive(String arrive) {
        this.arrive = arrive;
    }

    public double getPrix() {
        return prix;
    }

    public void setPrix(double prix) {
        this.prix = prix;
    }

    public String getType_reservation() {
        return type_reservation;
    }

    public void setType_reservation(String type_reservation) {
        this.type_reservation = type_reservation;
    }

    public int getCode_liv() {
        return code_liv;
    }

    public void setCode_liv(int code_liv) {
        this.code_liv = code_liv;
    }

    public int getNbr_place() {
        return nbr_place;
    }

    public void setNbr_place(int nbr_place) {
        this.nbr_place = nbr_place;
    }

    @Override
    public String toString() {
        String mp= "Reservation{" + "id_reservation=" + id_reservation + ", " + client.toString() + ", " + chauffeur.toString() + ", depart=" + depart + ", arrive=" + arrive + ", prix=" + prix + ", type_reservation=" + type_reservation;
        if(code_liv != 0){
             mp += ", code_liv=" + code_liv + ", nbr_place=" + nbr_place + '}';
        }
           if(type_reservation.equals("covoiturage")){
             mp += ", nbr_place=" + nbr_place + '}';
        }
           
           return mp;
    }

    @Override
    public int hashCode() {
        int hash = 5;
        hash = 61 * hash + this.id_reservation;
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Reservation other = (Reservation) obj;
        if (this.client.getId_user() != other.client.getId_user()) {
            return false;
        }
        if (!Objects.equals(this.depart, other.depart)) {
            return false;
        }
        if (!Objects.equals(this.arrive, other.arrive)) {
            return false;
        }
        if (!Objects.equals(this.type_reservation, other.type_reservation)) {
            return false;
        }
        return true;
    }
    
    
    
}
>>>>>>> ghada
